packages/web-twig/src/Resources/components/Select/stories/SelectValidationState.twig

```diff
@@ -0,0 +1,55 @@
+<section class="docs-Section">
+    <h2 class="docs-Heading">Select Validation State</h2>
+
+    <div class="docs-FormFieldGrid">
+        <Select
+            id="select-success"
+            name="select-success"
+            label="Validation success"
+            validationText="Success message"
+            validationState="success"
+            isRequired
+        >
+            <option value="" selected disabled>Placeholder</option>
+            <option value="1">Option 1</option>
+            <option value="2">Option 2</option>
+            <optgroup label="Sub options">
+                <option value="3">Option 3</option>
+                <option value="4">Option 4</option>
+            </optgroup>
+        </Select>
+        <Select
+            id="select-warning"
+            name="select-warning"
+            label="Validation warning"
+            validationText="Warning message"
+            validationState="warning"
+            isRequired
+        >
+            <option value="" selected disabled>Placeholder</option>
+            <option value="1">Option 1</option>
+            <option value="2">Option 2</option>
+            <optgroup label="Sub options">
+                <option value="3">Option 3</option>
+                <option value="4">Option 4</option>
+            </optgroup>
+        </Select>
+        <Select
+            id="select-danger"
+            name="select-danger"
+            label="Validation danger"
+            validationText="{{ ['Warning validation message', 'Second warning validation message'] }}"
```

Comment:
Isn't this Danger validation text actually? :) 

```suggestion
            validationText="{{ ['Danger validation text', 'Second danger validation text'] }}"
```

also it would be nice if it would be the same as in Web package...

Reply:
Fixed, But In the web package there is only 'Validation Text' and for the `success' status the text is completely missing, FYI